components/Onboarding/Section.js

```diff
@@ -0,0 +1,113 @@
+import React, { Fragment } from 'react'
+import { css } from 'glamor'
+
+import DoneIcon from 'react-icons/lib/md/done'
+
+import {
+  Button,
+  Loader,
+  mediaQueries,
+  colors
+} from '@project-r/styleguide'
+
+const HEADING_FONT_SIZE_MOBILE = 22
+const HEADING_FONT_SIZE_DESKTOP = 30
```

Comment:
As mentioned in `Page.js`: IMO use a space constant and just write the font size in place